src/main/java/frc/robot/swerve/ModuleIOReal.java

```diff
@@ -62,7 +62,7 @@ public void updateInputs(SwerveModuleInputs inputs) {
         driveStatorChargeUsedCoulomb.update(inputs.driveMotorStatorCurrent);
         inputs.driveMotorStatorCurrentOverTime = driveStatorChargeUsedCoulomb.get();
         inputs.driveMotorPosition = driveMotor.getRotorPosition().getValue();
-        inputs.driveMotorVelocity = getVelocity();
+        inputs.driveMotorVelocity = utils.units.Units.rpsToMetersPerSecond(getVelocity(), SwerveConstants.WHEEL_DIAMETER/2);
```

Comment:
A `Messure<Velocity>` should be used here and in line 101.

Reply:
What is that? I tried looking it up and couldn't find anything.

Reply:
It's the wpilib units library. It allows for easy unit conversion. [https://docs.wpilib.org/en/stable/docs/software/basic-programming/java-units.html](url)

Reply:
Oh I didn't know there was a new units library.